module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -2366,18 +2365,6 @@ protected function getItemStatusesForBib(string $id, bool $checkHoldings, ?array
         return $statuses;
     }
 
-    /**
-     * Extract the actual call number from item's call number field
-     *
-     * @param string $callNumber Call number field
-     *
-     * @return string
-     */
-    protected function extractCallNumber(string $callNumber): string
```

Comment:
This seems to be reverting changes from another recent Sierra PR. Is that intentional? It doesn't seem related to the intent of this one...

Reply:
Oops, I thought I took care of that. That's what you get for working with so many different versions. Should be fixed now.